docs/src/main/sphinx/installation/cli.rst

```diff
@@ -80,6 +80,35 @@ The detailed behavior is as follows:
   authentication system configuration.
 * Expired tokens force you to log in again.
 
+.. _cli-certificate-auth:
+
+Certificate authentication
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Use the following command line arguments to allow you to connect the CLI to a
+cluster that enables client :doc:`certificate authentication
+</security/certificate>`.
+
+.. list-table:: Certificate authentication options
+   :widths: 35 65
+   :header-rows: 1
+
+   * - Option
+     - Description
+   * - ``--keystore-path=<path>``
+     - Keystore path to a :doc:`PEM </security/inspect-pem>` or :doc:`JKS
+       </security/inspect-jks>` file, which must include a certificate and
+       private key for the CLI.
```

Comment:
How about breaking this up along these lines:

Path to a PEM or JKS  keystore file. The file must contain a certificate that is trusted by the Trino cluster you connect to.

Also .. relative or absolute path both work? And maybe add an example for a pem file in etc or wherever

Reply:
OK, but many questions remain. @dain, this can also point to a PKCS #12 file?